src/vendor/modules.txt

```diff
@@ -83,5 +86,5 @@ google.golang.org/grpc/stats
 google.golang.org/grpc/status
 google.golang.org/grpc/tap
 # gopkg.in/yaml.v2 v2.4.0
-## explicit
+## explicit; go 1.15
```

Comment:
Curious what the `; go 1.15` means here.

Reply:
no clue. maybe cause the go mod used to be 1.15 it's doing something different until we upgrade?

Reply:
Thanks! I still haven't made friends with the data in `modules.txt` so that's as good a guess as any I've got 😬